.nuget/packages.config

```diff
@@ -5,6 +5,6 @@
   <package id="MSBuild.Extension.Pack" version="1.5.0" />
   <package id="NuGet.CommandLine" version="3.5.0" />
   <package id="NuGet.for.MSBuild" version="1.4.3" targetFramework="net45" />
-  <package id="NUnit.Runners" version="2.6.3" />
+  <package id="NUnit.ConsoleRunner" version="3.11.1" />
   <package id="Remotion.BuildTools.MSBuildTasks" version="1.0.6180.28602" />
```

Comment:
There's an update missing.

Reply:
It might also be possible, that we no longer need those entries that already listed in build/packages.config. historically, we needed the .nuget/packages.config for solutionlevel-packages. Those then would not get registered at the project level. If we remove the entries and the package restore (starting from an empty packages folder) still works, we're good.